src/wgpu_renderer.rs

```diff
@@ -0,0 +1,478 @@
+// Copyright 2024 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
```

Comment:
Given that this new file is essentially a copy-paste of an old file, let's keep the old file's copyright year of 2022.

```suggestion
// Copyright 2022 the Vello Authors
// SPDX-License-Identifier: Apache-2.0 OR MIT
```